source/Government.cpp

```diff
@@ -510,3 +515,13 @@ bool Government::IsProvokedOnScan() const
 {
 	return provokedOnScan;
 }
+
+
+
+bool Government::IsUsingForeignPenaltiesFor(const Government *government) const
+{
+	for(const Government *gov : useForeignPenaltiesFor)
+		if(gov == government)
+			return true;
+	return false;
```

Comment:
```suggestion
	auto it = find(useForeignPenaltiesFor.begin(), useForeignPenaltiesFor.end(), government);
	if(it != useForeignPenaltiesFor.end())
	    return true;
	return false;
```

I think this might be faster, not sure though.
Micro enhancements XD